MANIFEST.in

```diff
@@ -1,24 +1,31 @@
 prune asset
```

Comment:
Looks like this line is the only entry that actually makes sense to be in this file.

Reply:
That is far from true, i tried this myself but the result caused problems in multiple places, one of them was the check-manifest. The other was that extra built docs were included and that desired json/yaml were not included from inside src/molecule. On the other hand if you put generic recursive includes for those two extensions you may endup getting stuff like `src/molecule/test/.mypy_cache/.../foo.json` which are excluded by default.

I know that testing behavior of Manifest.in is quite tricky as it is highly dependent on what you may have on disk, like temp files, stuff that are not VCS tracked. 

`check-manifest` command is very useful for debugging as it reports both kind of unexpected exclusions or inclusions but still fails to give correct recommendations or to spot the bug I mentioned.

Reply:
Oh, so your problem was because of the recursive inclusion and not a bug in `setuptools-scm`. That makes sense, then.